ihatemoney/models.py

```diff
@@ -262,6 +262,14 @@ def get_member_bills(self, member_id):
             .order_by(Bill.id.desc())
         )
 
+    def get_newest_bill(self):
+        """Returns the most recent bill (according to bill date) or None if there are no bills"""
```

Comment:
I find it weird to have two return types (None and a list), but that's probably not a problem ? :thinking:

Reply:
If we always return a list, we might want to get rid of the comments.

Reply:
It returns a single bill, not a list. Is there a way to clarify this? I mean, the function is called get_something_bill, without S :)

Reply:
Oh my bad, it was very clear but I… I don't know I was somewhere else I guess :sleeping: